167_Two_Sum_2_Input_array_is_sorted.py

```diff
@@ -0,0 +1,9 @@
+# time o(n^2)
```

Comment:
O (n^2) is too complex
avoid O(n^2)

If n is 10^3 or less, there is no problem, but please state so